snc.sh

```diff
@@ -113,6 +113,12 @@ function create_pvs() {
 # This follows https://blog.openshift.com/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms/
 # in order to trigger regeneration of the initial 24h certs the installer created on the cluster
 function renew_certificates() {
+    # Get the cli image from release payload and update it to bootstrap-cred-manager resource
+    echo ${OPENSHIFT_PULL_SECRET} > pull-secret
```

Comment:
Maybe we should look into storing a filename in OPENSHIFT_PULL_SECRET rather than the pull secret data, this would make this kind of operations nicer

Reply:
@cfergeau yes may be we will do that.